src/elm/Page/Dashboard/Analysis.elm

```diff
@@ -593,12 +593,7 @@ fetchAnalysis { accountName, selectedCommunity, shared } { profile, statusFilter
                     Present filterRecord
 
         args =
-            { input =
-                { symbol = Eos.symbolToString selectedCommunity
-                , account = Eos.nameToString accountName
-                , filter = filter
-                }
-            }
+            { communityId = Eos.symbolToString selectedCommunity }
```

Comment:
filter here keep existing, right?

Reply:
Yeah, the filter will be moved into optional arguments according to the new types. I was pretty rough at updating GraphQl bindings, for now, wanted to satisfy the compiler first. I still need to revisit these changes, now it's partially working. Just wanted to have a draft PR to share. I like the changes so far! Less code, more straightforward and secure flow. Especially signUp/signIn.